packages/core/useReducer/index.ts

```diff
@@ -0,0 +1,16 @@
+import { reactive, UnwrapRef } from 'vue-demi'
+
+export interface Action {
+  type: string
+  [propName: string]: any
+}
```

Comment:
Is this okay? Or do you think it should be more flexible?

Reply:
Just checked react's approach https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bbc8c7c7b7e92ba094ea349d56977e00f6f7f42d/types/react/index.d.ts#L927-L1006
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bbc8c7c7b7e92ba094ea349d56977e00f6f7f42d/types/react/index.d.ts#L871-L880

Which I think you can refer to :)

Reply:
Sorry, wasnt very motivated to do this in the last month and still havent figured out how i wanna do the reset. I'll close this for now, sorry

Reply:
Enjoy holiday and thanks for your awesome open source work !